MelonJs.Experiments/FileCrudApi/index.js

```diff
@@ -1,6 +1,11 @@
-load(__basedir + "/routes/getEntriesRoute.js")
-load(__basedir + "/routes/writeEntryRoute.js")
-load(__basedir + "/routes/deleteEntryRoute.js")
+const loadOptions = {
+    useUnsafeInjectorLoader: true,
+    useHttpRequest: false
+} 
```

Comment:
shouldn't it be abstracted on another layer?

Reply:
How?